05LessonFive/Checkpoint1/index.html

```diff
@@ -1,10 +1,24 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Landscape Design</title>
+  <title>Berries</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+  <div class="berryimg">
+    <img src="./img/berries.jpg">
+  </div>
+
+  <div class="berryparagraph">
+    <span><h2>Berries</h2>
```

Comment:
in general, span is used for a small subset of an element. could you take out the span and apply that css to the div instead?